website/static/js/osfHelpers.js

```diff
@@ -362,6 +362,15 @@ var mapByProperty = function(list, attr) {
 };
 
 
+/** 
+ * Replaces encoded & with decoded values
+ */
+var decodeText = function(text) {
+    if(typeof text === 'string') {
+        return text.replace(/&amp;/g, '&');
+    }
+    return text;
```

Comment:
Do we actually expect non-string input here? Or perhaps is this function being to forgiving to different kinds of input?